test-data/loop.yaml

```
@@ -266,3 +266,151 @@
+  - "r0.type=number"
+  - "r0.uvalue=100001"
+messages:
+  - "exit: Could not prove termination."
+
+---
+test-case: possible infinite loop
+options: ["termination"]
+
+pre: [r0.type=number]
+code:
+    <start>: |
```

yamllint

[error] wrong indentation: expected 2 but found 4

(indentation)